### ISSUE TITLE
Rosdep issue

### ISSUE THREAD

**Issue by ferpalacios** [tum-vision/tum_ardrone#31]
Hello!
I have followed the steps to install the package, but when I try to install dependencies with rosdep install tum_ardrone-indigo-devel I get the following error:
ERROR: Rosdep cannot find all required resources to answer your query
Missing resource tum_ardrone-indigo-devel
ROS path [0]=/opt/ros/indigo/share/ros
ROS path [1]=/home/fernando/ros/stacks
ROS path [2]=/home/fernando/catkin_ws/src
ROS path [3]=/opt/ros/indigo/share
ROS path [4]=/opt/ros/indigo/stacks

I am using indigo version of ROS and I have tried with indigo-devel and hydro-devel branches

**Comment by rohanbhargava11** [comment]
I will try installing tum_ardrone on Indigo by today evening. If everything builds fine I will send you the updated instructions.

**Comment by ferpalacios** [comment]
I just have fixed the error. Instead of use "ardrone_autonomy" from tum-vision, I have used the one from AutonomyLab (https://github.com/AutonomyLab/ardrone_autonomy/tree/indigo-devel). I have installed both (ardrone_autonomy and tum_ardrone) on catkin workspace and then I did catkin_make. I really didn't needed rosdep because all dependencies were installed. I don't know if using AutonomyLab version with tum_ardrone could cause problems in the future, but It looks to work well.
If anyone has issues with rosdep, I suggest to use rospack depends1 tum_ardrone and then try to install manually those packages.
Anyway, thanks for the help.
